orchestrator-salt/src/main/resources/salt-common/salt/telemetry/upgrade.sls

```diff
@@ -0,0 +1,37 @@
+{%- from 'telemetry/settings.sls' import telemetry with context %}
+{% if salt['pillar.get']('cloudera-manager:paywall_username') %}
+  {% set paywall_username = salt['pillar.get']('cloudera-manager:paywall_username') %}
+  {% set paywall_password = salt['pillar.get']('cloudera-manager:paywall_password') %}
+  {% set curl_cmd = 'curl --max-time 30 -s -k -f -u $PAYWALL_USERNAME:$PAYWALL_PASSWORD ' + telemetry.repoGpgKey %}
```

Comment:
have you checked if paywall user/pw would show up in our logs if you use it this way?

Reply:
not in our CB logs but it's in the salt logs regarding the environment, not sure that is a problem, so i can change this to read the values from the repo file, then it will be totally hidden

Reply:
I think paywall password is sensitive data